modules/core/src/test/java/org/apache/ignite/internal/client/thin/ServicesTest.java

```diff
@@ -403,6 +403,31 @@ public void testServiceDescriptors() throws Exception {
         }
     }
 
+    /** Test to verify service failover after a node failure. */
+    @Test
+    public void testServiceFailoverAfterNodeFailure() throws Exception {
+        String TEST_SRVC_NAME = "cluster-singleton-svc";
```

Comment:
Actually, you can use existing singleton service instead - CLUSTER_SINGLTON_SERVICE_NAME with TestServiceInterface.